db/migrate/20211109152523_devise_create_users.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+class DeviseCreateUsers < ActiveRecord::Migration[6.1]
+  def change
+    create_table :users do |t|
+      ## Database authenticatable
+      t.string :email,              null: false, default: ""
+      t.string :encrypted_password, null: false, default: ""
+      t.string :username,           null: false
```

Comment:
Looks like your model is expecting this username to be unique - we should handle that here on the DB as well (one of the few constraints we can use on the DB!)